configs/sst_insights.yaml

```diff
@@ -8,19 +8,56 @@ data:
   packages: []
 
   package_placeholders:
-    insights-client:
-      srpm: insights-client
-      description: insights
-      requires:
-      - coreutils
-      - gpg
-      - pciutils
-      - python3-magic
-      - python3-six
-      - python3-requests
-      - systemd
-      - python3-PyYAML
-      - tar
+    - srpm_name: insights-client
+      rpms:
+      - rpm_name: insights-client
+        description: insights-client
+        dependencies:
+        - coreutils
+        - gpg
+        - pciutils
+        - policycoreutils-python-utils
+        - python3-PyYAML
+        - python3-magic
+        - python3-requests
+        - python3-setuptools
+        - python3-six
+        - systemd
+        - tar
+
+    - srpm_name: rhc
+      rpms:
+      - rpm_name: rhc
+        description: rhc
+        dependencies:
+        - policycoreutils-python-utils
+        - yggdrasil
+        - yggdrasil-worker-package-manager
+
+    - srpm_name: rhc-worker-playbook
+      build_dependencies:
+      - rhc
+      rpms:
+      - rpm_name: rhc-worker-playbook
+        description: rhc-worker-playbook
+        dependencies:
+        - ansible-core
+        - insights-client
+        - python3-PyYAML
+        - python3-requests
+
+  arch_packages:
+    aarch64:
+    - yggdrasil-worker-package-manager
+
+    ppc64le:
+    - yggdrasil-worker-package-manager
+
+    s390x:
+    - yggdrasil-worker-package-manager
+
+    x86_64:
+    - yggdrasil-worker-package-manager
```

Comment:
`arch_packages` is for when a package is only for some architectures.  (It is also not needed just for excluding i686.)  What are you excluding here?

Reply:
ah ok; from what I remember in the past (during the el9 work), `packages` was for noarch packages and `arch_packages` was for arch-specific ones

in this case, I can also fold the `arch_packages` in `sst_subscription_manager-subscription-manager.yaml` to `packages`